crates/bevy_scene/src/lib.rs

```diff
@@ -40,9 +41,9 @@ impl Plugin for ScenePlugin {
             .add_asset::<Scene>()
             .init_asset_loader::<SceneLoader>()
             .init_resource::<SceneSpawner>()
-            .add_systems(Update, scene_spawner_system)
+            .add_systems(PreUpdate, scene_spawner_system)
             // Systems `*_bundle_spawner` must run before `scene_spawner_system`
-            .add_systems(PreUpdate, scene_spawner);
+            .add_systems(PreUpdate, scene_spawner.before(scene_spawner_system));
```

Comment:
One minor suggestion:
```suggestion
            .add_systems(PreUpdate, (scene_spawner, scene_spawner_system).chain());
```

Reply:
Pushed a commit which keep the comment.